docs/reference/search/search-your-data/semantic-search-inference.asciidoc

```diff
@@ -6,9 +6,11 @@
 
 The instructions in this tutorial shows you how to use the {infer} API with
 various services to perform semantic search on your data. The following examples 
-use Cohere's `embed-english-v3.0` model and OpenAI's `text-embedding-ada-002`
-second generation embedding model. You can use any Cohere and OpenAI models,
-they are all supported by the {infer} API.
+use Cohere's `embed-english-v3.0` model, the `all-mpnet-base-v2` model from
+HuggingFace, and OpenAI's `text-embedding-ada-002` second generation embedding
+model. You can use any Cohere and OpenAI models, they are all supported by the
+{infer} API. For a list of supported models available on HuggingFace, refer to
+{ml-docs}/ml-nlp-model-ref.html[the compatible third party NLP models] page.
```

Comment:
Hmm, I think the models that we support for hugging face via the inference api is different than what we have listed in that `ml-nlp-model-ref` page.

The ones I've tested are these:

https://huggingface.co/sentence-transformers/all-MiniLM-L12-v2
https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2
https://huggingface.co/intfloat/multilingual-e5-base
https://huggingface.co/intfloat/e5-base-v2
https://huggingface.co/sentence-transformers/all-mpnet-base-v2
https://huggingface.co/intfloat/e5-small-v2
https://huggingface.co/intfloat/multilingual-e5-small

Others _may_ work, but we haven't tested them. And they'd need to adhere to the response format that we're expecting, so we should probably list these specific models somewhere as the ones we support right now.

Reply:
Precious info, thank you! In an upcoming commit, I'll add the list to the PUT inference API page as part of this PR.